README.md

```diff
@@ -115,6 +115,18 @@ Executing tests that do not pattern match "System"
 npm test -- --ver=alpha --invert --grep="System"
 ```
 
+## Test with core
+
+```bash
+CORE_PATH=*path to core* npm run core
+```
+
+or without specifying the core path (core will be cloned from GH into parent folder):
```

Comment:
It might be better to clone it into a subdirectory. I'm sure I'm not alone keeping my repos in the same parent directory, and if I'm not specifying a local copy to use it means I want latest develop